model_compression_toolkit/core/common/statistics_correction/compute_activation_bias_correction_of_graph.py

```diff
@@ -0,0 +1,210 @@
+                                       kernel_size: str) -> Graph:
+    """
+    Compute the activation bias correction term, and store it in the final activation
+    quantization configuration.
+
+    Args:
+        graph: Graph with nodes to compute the activation bias correction for each node's final activation quantization configuration.
+        core_config: Configuration object containing parameters of how the model should be quantized.
+        fw_info: Framework info like lists of nodes their kernel should quantized.
+        fw_impl: FrameworkImplementation object with a specific framework methods implementation.
+        linear_node: Node to compute the activation bias correction for.
+        prev_node: Node to compute the activation error caused by his activation quantization.
+        kernel_size: The framework specific attribute name of the convolution layer's kernel size.
+
+    Returns:
+        Graph with activation bias correction term for each node.
+    """
+
+    # Check if 'kernel_size' is a key in the framework-specific attributes of the linear_node, if it is then the
+    # linear_node is a convolution
+    if kernel_size in linear_node.framework_attr.keys():
+        # Retrieve the value of 'kernel_size' and check if it is not 1 or (1, 1). This feature supports only kernel
+        # size of 1 or (1, 1).
+        if linear_node.framework_attr.get(kernel_size) not in [1, (1, 1)]:
+            # If the kernel size is not 1 or (1, 1), return the current graph unmodified
+            return graph
+
+    prev_node_act_quant_cfg = prev_node.final_activation_quantization_cfg
+
+    # Check if the previous node's has activation quantization configuration and if the previous node have the
+    # histogram collector
+    if prev_node_act_quant_cfg is None or not hasattr(graph.get_out_stats_collector(prev_node), 'hc'):
+        return graph  # pragma: no cover
+
+    float_bins, float_count = graph.get_out_stats_collector(prev_node).hc.get_histogram()
+
+    # Calculate the centers of the float bins
+    float_centers = calculate_bin_centers(float_bins)
+
+    # Quantize the bin edges and calculate the centers of the quantized bins
+    quant_bins = prev_node_act_quant_cfg.quantize_node_output(fw_impl.to_tensor(float_bins))
+    quant_bins = fw_impl.to_numpy(quant_bins)
+    quant_centers = calculate_bin_centers(quant_bins)
+
+    # Calculate the mean of the both the float and the quantized bin centers, weighted by the bin counts
+    mean_float_centers = np.sum(float_centers * float_count) / np.sum(float_count)
+    mean_quant_centers = np.sum(quant_centers * float_count) / np.sum(float_count)
+
+    # Compute the difference between the mean quantized center and the mean float center
+    mean_diff = mean_quant_centers - mean_float_centers
+
+    # Check if activation bias correction is enabled based on the configured threshold
+    if core_config.quantization_config.activation_bias_correction_threshold > 0:
```

Comment:
this seems wrong. init the threshold to inf if you want it to be enabled by default.